bulk/eng_bulk_load-AssetRecord.csv

```diff
@@ -122,465 +128,481 @@ CGCON-CWFPCP-10302,A00847.1,Sensor,0,Coastal Wire-Following Profiler Controller,
+CGCON-MCPM03-00020,,Sensor,0,Multi-Function Node Communications and Power Manager,Communications and Power Manager,WHOI,CPM,20,,,,,
+CGCON-MCPM03-50001,OL000280,Sensor,0,Multi-Function Node Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0001,,,,CP04OSSM-00004-CPM3 TO BE DELETED,
+CGCON-MCPM03-50004,OL000262,Sensor,0,Multi-Function Node Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0004,,,,CP03ISSM-00004-CPM3 TO BE DELETED,
+CGCON-MCPM03-50005,OL000230,Sensor,0,Multi-Function Node Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0005,,,,CP01CNSM-00005-CPM31 TO BE DELETED,
+CGCON-MDCL35-00001,OL000444,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,3703-00292-0001,,,,CE07SHSM-00003-DCL35,Oregon State University
+CGCON-MDCL35-00002,OL000184,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2,,,,CE01ISSM-00001-DCL35,Oregon State University
+CGCON-MDCL35-00003,OL000490,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0003,,,,CP01CNSM-00001-DCL35,Woods Hole Oceanographic Institution
+CGCON-MDCL35-00005,,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,5,,,,CE01ISSM-00002-DCL35,Oregon State University
+CGCON-MDCL35-00006,,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,6,,,,CE07SHSM-00004-DCL35,Oregon State University
+CGCON-MDCL35-00007,OL000504,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0007,,,,CP01CNSM-00002-DCL35,Woods Hole Oceanographic Institution
+CGCON-MDCL35-00008,OL000235,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0008,,,,CP01CNSM-MFD35-00-DCLENG000 - 5 & 6,Woods Hole Oceanographic Institution
+CGCON-MDCL35-00009,OL000537,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0009,,,,CP04OSSM-00001-DCL35,Woods Hole Oceanographic Institution
+CGCON-MDCL35-00010,OL000404,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,3703-00292-0010,,,,CE06ISSM-00004,Oregon State University
+CGCON-MDCL35-00011,OL000196,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,11,,,,CE01ISSM-00003-DCL35,Oregon State University
+CGCON-MDCL35-00012,OL000285,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0012,,,,"CP04OSSM-MFD35-00-DCLENG000,4",Woods Hole Oceanographic Institution
+CGCON-MDCL35-00013,OL000267,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0013,,,,"CP03ISSM-MFD35-00-DCLENG000,4",Woods Hole Oceanographic Institution
+CGCON-MDCL35-00014,,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,14,,,,CE01ISSM-00006-DCL35,Oregon State University
```

Comment:
According to eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet the Manuf Serial No. should be updated to 0014

Reply:
I don't want change OSU S/Ns, that's up to them. I got a little overzealous in the Spreadsheet.